pkgs/development/web/kore/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, callPackage, openssl }:
+
+let
+  source = callPackage ./source.nix { };
```

Comment:
Why did you put source in a separate file? Generally this is only accepted when it must be updated by tools that cannot update `default.nix`.